### PR TITLE
Ideal nav merge search bar

### PR DESCRIPTION
**Pull request by kosmydel**
Screenshot 2023-12-20 at 10 20 07
Screenshot 2023-12-20 at 14 09 39

### CODE REVIEW

File: src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/TopBar.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
```
Comment (MaciejSWM) [inline src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/TopBar.js:1]: Can TopBar.js be written in TS?

File: src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/TopBar.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+        Navigation.navigate(ROUTES.SEARCH);
+    }, [isCreateMenuOpen]);
+
+    return (
+        <View
+            style={[styles.gap6, styles.flexRow, styles.ph5, styles.pv3, styles.justifyContentBetween, styles.alignItemsCenter]}
+            dataSet={{dragArea: true}}
+        >
+            <PressableWithFeedback accessibilityRole={CONST.ACCESSIBILITY_ROLE.BUTTON}>
```
Comment (MaciejSWM) [inline src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/TopBar.js:35]: `accessibilityRole` is be deprecated. We should use `role=CONST.ROLE.BUTTON`

File: src/styles/theme/themes/dark.ts [hunk 3]
```diff
@@ -45,7 +45,7 @@ const darkTheme = {
     hoverComponentBG: colors.productDark300,
     activeComponentBG: colors.productDark400,
     signInSidebar: colors.green800,
-    sidebar: colors.productDark200,
+    sidebar: colors.productDark100,
```
Comment (MaciejSWM) [inline src/styles/theme/themes/dark.ts:48]: Probably same needs to be done to `light.js` theme

File: src/styles/utils/spacing.ts [hunk 4]
```diff
@@ -377,6 +377,10 @@ export default {
         paddingHorizontal: 24,
     },
 
+    ph7: {
```
Comment (MaciejSWM) [inline src/styles/utils/spacing.ts:380]: Is this used anywhere?

File: src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/TopBar.js (hunk 1 above)
Comment (kosmydel) [reply src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/TopBar.js:1]: I don't think so, the `SignInOrAvatarWithOptionalStatus` is written in JS.

File: src/styles/theme/themes/dark.ts (hunk 3 above)
Comment (kosmydel) [reply src/styles/theme/themes/dark.ts:48]: Great catch!

File: src/styles/utils/spacing.ts (hunk 4 above)
Comment (kosmydel) [reply src/styles/utils/spacing.ts]: No, I must have copied it from a different PR.

**Pull request merged**